### PR TITLE
Errors in InstructorCourseDetailsPageUiTest #4636

### LINKED ISSUE #4636: Errors in InstructorCourseDetailsPageUiTest
- Line 177:
```java
String alicePassword = TestProperties.inst().TEST_STUDENT2_PASSWORD;
```
but Alice uses student 1 account. (!!!!)
- Try doing this: allow email access for Charlie, but _not_ for Alice. The test will still (falsely) pass.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #4636 
Straightforward but crucial fixes. Anyone welcome to review.

### CODE REVIEW

File: src/test/java/teammates/test/driver/EmailAccount.java [hunk 1]
```diff
@@ -26,32 +26,27 @@
...
+        for (int i = messages.length - 1; i >= messages.length - maxEmailsToCheck; i--) {
+            Message message = messages[i];
+
+            if (isRegistrationEmail(message, courseId)) {
+                String body = getEmailBody(message);
+                String key = getKey(body);
+                message.setFlag(Flags.Flag.SEEN, true);
+
+                inbox.close(true);
+                return key;
```
Comment (damithc) [inline src/test/java/teammates/test/driver/EmailAccount.java]: What's the change here?

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseDetailsPageUiTest.java [hunk 2]
```diff
@@ -174,7 +174,7 @@ public void testRemindAction() {
         
         //Alice is already registered
         StudentAttributes alice = testData.students.get("CCDetailsUiT.alice.tmms@CCDetailsUiT.CS2104");
-        String alicePassword = TestProperties.inst().TEST_STUDENT2_PASSWORD;
+        String alicePassword = TestProperties.inst().TEST_STUDENT1_PASSWORD;
```
Comment (damithc) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseDetailsPageUiTest.java]: Can we change alice, charlie etc. to student1, student2?

File: src/test/java/teammates/test/driver/EmailAccount.java (hunk 1 above)
Comment (wkurniawan07) [inline src/test/java/teammates/test/driver/EmailAccount.java]: Got rid of the try-catch block, which erroneously "swallowed" crucial exceptions that might be thrown due to the security issues and what-nots. That reduces the indentation level of everything inside by 4 spaces. If you review with `?w=1` you'll notice that nothing inside the block is touched.

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseDetailsPageUiTest.java (hunk 2 above)
Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseDetailsPageUiTest.java]: It's used in more places than just this particular part of the test. I guess better leave it to first timer/contributor to fix the entire file for good.

**Comment by wkurniawan07** [comment]
@kanghj since no one is volunteering to review, I have to arrow :p

**Comment by kanghj** [comment]
Looks good to me. Remember to make an issue to rename the students.

**Pull request merged**